### DISCUSSION TITLE
Problems connecting to Chromecast Audio device

### DISCUSSION THREAD

**Discussion by TookieClothespin** [SamDel/ChromeCast-Desktop-Audio-Streamer#155] [Q&A]
Has anyone used this app with the old Google Chromecast Audio device?  They were the little hockey-puck devices that google abandoned.  I have two and was trying to see if they would work with this app.  The log show a lot of timeout issues.

![chromecastAudio](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/assets/152813755/e93ede19-3e15-479c-a748-58c7e9267bc5)

**Comment by FA-Bubba** [comment]
Yes, I have several...  I bought a bunch when Google ceased production.  Here's how I use them:

I bought 8 pairs of Philips 2.0 Multimedia Speakers, (SPA4270BT/37), and connect them via the Google CCA devices, which broadcasts stereo through the speakers.  I also have 1 Google Home Hub, and 5 of the Google Home Minis, for a total of 14 "Devices" in the Desktop Audio Steamer display.  I am casting from iTunes, and have several PlayLists to select for various moods.

No problems with the Google CCAs.

**Reply by TookieClothespin** [reply]
As a test, I deleted my CCA and re-added it via Google Home app and now the CCA is working nicely with ChromeCast Desktop Audio Streamer.
My CCA had been powered off for months.  I wonder if when reconnecting to power the CCA needs to be deleted/readded because its old IP may have been reassigned in the network to something else?  Maybe I need to give a static IP to the CCA?  I do not have a google home hub in my system.   Thanks for your feedback.

**Comment by FA-Bubba** [comment]
I have assigned static IPs to ALL of my speakers & devices.
